multithreading-demo/README.md

```diff
@@ -33,21 +33,32 @@ The plugin will also gather the diagnostic information during the native image b
 </plugin>
 ```
 
-To build and run the projects, you can either use [GraalVM](https://www.graalvm.org/downloads/), or any other JDK distribution.
+To build and run the projects, you can either use [GraalVM JDK Downloader](https://github.com/graalvm/graalvm-jdk-downloader), or any other JDK distribution.
 
-## Multithreading Demo Oversized
+## Preparation
 
-1. Download or clone the repository and navigate into the _multithreading-demo/multithreading-demo-oversized_ directory:
+1. Download and install the latest GraalVM JDK with Native Image and Java on Truffle (Espresso) components using the [GraalVM JDK Downloader](https://github.com/graalvm/graalvm-jdk-downloader).
```

Comment:
You don't need  Java on Truffle (Espresso)  for this demo. Remove in text and update the command.